src/objects/Points.js

```diff
@@ -8,6 +8,7 @@ import { BufferGeometry } from '../core/BufferGeometry.js';
 
 /**
  * @author alteredq / http://alteredqualia.com/
+ * @author Lewy Blue / https://discoverthreejs.com/
```

Comment:
FYI - We do not add authorship for minor changes, otherwise the authors on some files would be quite extensive.
https://github.com/mrdoob/three.js/pull/13332#pullrequestreview-96758707

Reply:
I added my name since this is adding a new feature, but I'm ok either way.

Reply:
@WestLangley the code I added here makes up around 25% of the file, compared to the other PR you linked where the code added is just a couple of %. Can you state more clearly at what point you think it's appropriate to add authorship/maintership?

Reply:
If you created the file, you are the author. If you edited it, you are an editor.

You can always include a link -- as many have -- in the title of the examples you create. :-)

Obviously, there is not a metric we can use. Err on the side of modesty. We want to be fair to everyone. 

Also, recognition for all contributions is given in the release docs.

Reply:
That's fine, I guess I was thinking this more of putting myself as a "maintainer" than "author", to use the nomenclature from the other PR you linked. 

Obviously, I'm not trying to claim ownership of this file - the code I added here is largely copied from `Mesh` anyway. My thinking was that it's useful for people to know who to blame/ask for help by looking at the top of the file. 

 However, if you're OK with it, I will add links to the examples I've created since that will give me that all important SEO juice 😁

Reply:
I don't make the final decisions, but IMO, that is a better approach. AFAIK, there has been no objection to adding links to the examples you create.